Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -668,10 +669,11 @@ void DrawMol::extractBrackets() {
         }
         annotations_.emplace_back(da);
       }
-      std::string label;
-      if (sg.getPropIfPresent("LABEL", label)) {
+      // function to draw a label at the bottom of the bracket.
```

Comment:
That's a lot of code for a lambda. Could this be a function in an anonymous namespace instead?

Reply:
It seemed like it was only relevant for this function, so I thought I’d make it a lambda so as not to bloat the number of functions in the global namespace. But you’re right, it is a fair few lines of code.  I can move it out into the main body if you think it would be tidier. I guess that it does make the enclosing function break the rule of thumb that functions should be fully visible in the screen in one go, except that I have a large screen and a small font so it does for me 😀.

Reply:
you can define in an an anonymous namespace just above this function... no crowding!
And you already have a name for it. :-)